plugin/pulpcore/plugin/stages/api.py

```diff
@@ -23,6 +23,56 @@ class Stage:
         """
         raise NotImplementedError(_('A plugin writer must implement this method'))
 
+    @staticmethod
+    async def batches(in_q, minsize=1):
+        """
+        Asynchronous iterator yielding batches of :class:`DeclarativeContent` from `in_q`. The
```

Comment:
Consider adding a single sentence description at the top of this docstring. Some docstring parsers use the single-top line meaningfully.

Reply:
done